setup.py

```diff
@@ -47,8 +47,9 @@ def get_version():
     author='Peter Waller (Thanks to Christopher Jones and Stefano Rivera)',
     author_email='p@pwaller.net',
     url='https://github.com/pwaller/pyfiglet',
-    packages=['pyfiglet', 'pyfiglet.fonts'],
-    package_data={'pyfiglet.fonts': ['*.flf', '*.flc']},
+    packages=['pyfiglet'],
+    package_data={'pyfiglet.fonts-standard': ['*.flf', '*.flc'],
+                  'pyfiglet.fonts-contrib': ['*.flf', '*.flc']},
```

Comment:
Does this really work?  `pyfiglet.fonts-standard` is not a valid python package, so theses lines are strange.

Reply:
I had added a note to hold off on this PR, but it got merged. I'm thinking of fixing this by merging the standard and contrib folders together, since that is what the Makefile does. I discovered the Makefile after working on the PR.